src/pages/patch/ParametersModal.tsx

```diff
@@ -1,35 +1,50 @@
-import React from "react";
+import React, { useState } from "react";
 import styled from "@emotion/styled";
 import Badge from "@leafygreen-ui/badge";
 import { Modal } from "components/Modal";
-import { Parameter } from "types/patch";
+import { StyledLink } from "components/styles";
+import { P2 } from "components/Typography";
+import { Parameter } from "gql/generated/types";
 
 interface ParametersProps {
   parameters: Parameter[];
   dataCy: string;
-  closeModal: () => void;
-  visible: boolean;
 }
+
 export const ParametersModal: React.FC<ParametersProps> = ({
   parameters,
   dataCy,
```

Comment:
nit. If we don't expect this component to be reused we can just declare the `data-cy` inline on line 36 instead of passing it in.